src/main/java/com/partymusicapp/Controllers/FollowController.java

```diff
@@ -0,0 +1,25 @@
+package com.partymusicapp.Controllers;
+
+import com.partymusicapp.models.Follow;
+import com.partymusicapp.models.User;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping(value = "/pma/api/v1/follow")
+public interface FollowController {
+
+    @PostMapping(value = "/addFollower", consumes = MediaType.APPLICATION_JSON_VALUE)
+    Follow addFollow(@RequestParam String followerUsername, @RequestParam String followedUsername);
+
+
+    @DeleteMapping(value ="/removeFollower/{followerUsername}")
+    void removeFollower(@PathVariable User followerId);
```

Comment:
Ar trebuie sa fie numite AddFollow(am schimbat eu) si removeFollow. Ideea ii cand stergi un follow trebuie sa stii si follower-ul si followed cine ii, ca altfel o sa stergi toate follow-urile de la follower sau de la followed. Deci trebuie sa mai ai o variabila. Inca o chestie: II cam greu sa trimiti un user in request, trebuie sa il bilduiesti in frontedn, sa il chemi dupa id/username prima data si a.s.a. Schimba endpointul asta sa dai numa un String cu username cum am facut eu la addFollow. II mult mai user de facut front pe asta si de testat